app/helpers/cases_helper.rb

```diff
@@ -328,4 +328,15 @@ def show_escalation_deadline?(kase)
       kase.has_attribute?(:escalation_deadline) &&
       kase.within_escalation_deadline?
   end
+
+  def get_sar_recipient_label(recipient)
```

Comment:
Might be worth extracting these out into the `en.yml`

Reply:
Great spot Will, done :-)